doc/examples/transform/plot_tps_deformation.py

```diff
@@ -0,0 +1,148 @@
+r"""
+==========================================
+Interpolate images with thin-plate splines
+==========================================
+
+Thin-plate splines (TPS) refer to a method for interpolating data [1]_.
+In an image context, we use a 2D generalization of TPS, i.e., a two-dimensional
+simulation of a 1D cubic spline, which is the solution of the biharmonic equation [2]_.
+According to [3]_, the basic solution of the biharmonic function was expanded.
+Given corresponding source and target control points, TPS is used to transform a space.
+In this case our space is a 2D image.
+
+This script shows how to use the Thine Plate Spine Transformation for
+image warping.
```

Comment:
Please improve! "The examples presented here...?" Define/introduce image warping?

Reply:
I removed the lines as suggest [here](https://github.com/scikit-image/scikit-image/pull/7040#discussion_r1431748120).